nodegroups.tf

```diff
@@ -87,6 +87,22 @@ resource "aws_launch_template" "eks_node_groups" {
     enabled = true
   }
 
+  dynamic "instance_market_options" {
+    for_each = each.value.spot_nodes_enabled != null && each.value.spot_nodes_enabled == true ? ["do it"] : []
+    content {
+      market_type = "spot"
+
+      spot_options {
```

Comment:
@qedgardo couldn't we remove the `lookup()` function just for the dot naming?, i mean, for instance, doing this
`each.value.spot_options.block_duration_minutes` instead of this `lookup(each.value.spot_options, "block_duration_minutes", null)` ?

Reply:
I've used the lookup function to have a default value in case of missing the attribute